_projects/eff.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: "eff"
+category: "effects"
+description: "Extensible effects are an alternative to monad transformers for computing with effects in a functional way. This library is based on the “free-er” monad and an “open union” of effects described by Oleg Kiselyov in “Freer monads, more extensible effects“"
```

Comment:
I fixed the closing quote. Do you think it is better to use ASCII quotes? And do you know if they are escaped with `\` in that case?